index.html

```diff
@@ -113,15 +113,15 @@ <h2>Contact Form Project</h2>
             </div>
         </div>
         <!-- End New Project Card -->
-        <!-- Start Github -->
-               <div class="card">
-                <img src="contact-form/asset/contact.jpg" alt="new_project screenshot">
-                <div class="card-content">
-                    <h2>ABC</h2>
-                    <p>SDFHdkfdh</p>
-                    <a href="./contact-form/contact-form.html">View project</a>
-                </div>
+        <!-- Start Digital Clock -->
+        <div class="card">
+            <img src="contact-form/asset/contact.jpg" alt="new_project screenshot">
+            <div class="card-content">
+                <h2>Digital Clock</h2>
+                <p>Shows you the recent time digitally.</p>
```

Comment:
Please remove the start digital clock comment.

Reply:
changes made as per requested.